src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square implements Figure {
+    private final int side;
+    private final String color;
```

Comment:
Color is common feature of all figures and should be moved to parent abstract class for this all all other figures